src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -765,23 +751,15 @@ private static string RetrieveProcessUserName(Process process)
                 }
 
                 var tokenUser = Marshal.PtrToStructure<Win32Native.TOKEN_USER>(tokenUserInfo);
-
-                // Max username is defined as UNLEN = 256 in lmcons.h
-                // Max domainname is defined as DNLEN = CNLEN = 15 in lmcons.h
-                // The buffer length must be +1, last position is for a null string terminator.
-                int userNameLength = 257;
-                int domainNameLength = 16;
-                Span<char> userNameStr = stackalloc char[userNameLength];
-                Span<char> domainNameStr = stackalloc char[domainNameLength];
-                Win32Native.SID_NAME_USE accountType;
-
-                // userNameLength and domainNameLength will be set to actual lengths.
-                if (!Win32Native.LookupAccountSid(null, tokenUser.User.Sid, userNameStr, ref userNameLength, domainNameStr, ref domainNameLength, out accountType))
+                SecurityIdentifier sid = new SecurityIdentifier(tokenUser.User.Sid);
+                try
                 {
-                    return null;
+                    return sid.Translate(typeof(System.Security.Principal.NTAccount)).Value;
```

Comment:
Does `.Value` returns the user name in the format of `<domain-or-machine-name>\<user-name>`?

Reply:
It will be the Netlogon format `DOMAIN\user` just like the original code was doing.